_data/harnesses/produce-consume-lang/scala.yml

```
@@ -0,0 +1,261 @@
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/1_2-start-docker-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+
+    - title: Initialize the project
+      content:
+        - action: make_file
+          file: build.sbt
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_0-create-build-file.adoc
+
+        - action: make_file
+          file: project/build.properties
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_1-create-properties-file.adoc
+
+        - action: make_file
+          file: project/Dependencies.scala
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_2-create-deps-file.adoc
+
+        - action: make_file
+          file: project/plugins.sbt
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_3-create-plugins-file.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/sbt-extras.sh
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_4-sbt-extras.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-resources-dir.sh
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_5-resources-dir.adoc
+
+        - action: make_file
+          file: src/main/resources/application.conf
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_6-add-config-file.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-sources-dir.sh
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_5-sources-dir.adoc
+
+        - action: make_file
+          file: src/main/scala/io/confluent/developer/Configuration.scala
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_7-add-config-parser.adoc
+
+        - action: make_file
+          file: src/main/resources/logback.xml
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/2_8-add-logging-conf.adoc
+
+    - title: Create a schema for the events
+      content:
+        - action: execute
+          file: tutorial-steps/dev/make-schema-package.sh
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/3_0-schema-package.adoc
+
+        - action: make_file
+          file: src/main/scala/io/confluent/developer/schema/Book.scala
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/3_1-schema-case-class.adoc
+
+        - action: make_file
+          file: src/main/scala/io/confluent/developer/schema/BookType.scala
+          render:
+            file: tutorials/produce-consume-lang/scala/markup/dev/3_3-schema-case-class.adoc
+
+    - title: Setup the serialisation method
```

I circulated a question to the DevRel team about serialisation vs serialization and the consesus was to stick w/ the "z" spelling.  I'm going to make suggested edits for these throughout. 
```suggestion
    - title: Setup the serialization method
```